scripts/entrypoint.sh

```diff
@@ -2,6 +2,16 @@
 
 echo "starting jupyter notebook"
 
+if [ -n "$SPARK_DRIVER_HOST" ]; then
+    echo "Setting spark.driver.host to $SPARK_DRIVER_HOST"
+    source /opt/bitnami/scripts/spark-env.sh
```

Comment:
This is to get the conf file env var?

Reply:
yea. I think this is the script bitnami is using to load all env variables. It only has `export` commands.

Reply:
:+1: